### PR TITLE
COMMERCE-5608 Order workflow not reflected in the component

### PR DESCRIPTION
**Pull request by gianmarcobrunialti**

### CODE REVIEW

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/internal/model/WorkflowStatusModel.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.frontend.taglib.internal.model;
+
+/**
+ * @author Gianmarco Brunialti Masera
+ */
+public class WorkflowStatusModel {
```
Comment (marco-leo) [inline modules/apps/commerce/commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/internal/model/WorkflowStatusModel.java:20]: @gianmarcobrunialti don't you need to delete OrderStatusModel class?

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

**Comment by gianmarcobrunialti** [comment]
Tests keep failed because the cloud instance where the Jenkins build was run was offline :( Re-launching...

**Review by ivicac — APPROVED** [review]

**Comment by ivicac** [comment]
ci:forward

**Pull request closed without merging**